src/patternfly/components/NotificationDrawer/notification-drawer-task-group-toggle.hbs

```diff
@@ -0,0 +1,11 @@
+<button class="pf-c-notification-drawer__task-group-toggle{{#if notification-drawer-task-group-toggle--modifier}} {{notification-drawer-task-group-toggle--modifier}}{{/if}}"
```

Comment:
This markup is invalid. The children of a button should be `<span>` elements, and you can't have a link in a button.

<img width="633" alt="Screen Shot 2021-08-04 at 5 42 51 PM" src="https://user-images.githubusercontent.com/35148959/128264811-5b0355d5-ca9c-400b-84dd-ddca50cf6c40.png">

Reply:
@mcoker This is mirroring the existing group toggle, which includes `div`s. So I guess they all should be updated?

The link in the design for the task group toggle further complicates matters. This renders in Chrome, FF, and Safari, but I agree it is invalid according to the spec. What is your suggestion here?

Reply:
> This is mirroring the existing group toggle, which includes divs. So I guess they all should be updated?

Yeah I think so, or we can create a follow up issue to do the notification drawer items since it's out of scope for this PR. Up to you.

I think in this case, we might make the group toggle a div, and handle the click event in the react component, and keep any other clicks events on clickable elements in the toggle from bubbling up. WDYT @kmcfaul @nicolethoen?